Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/MousePointer.cs

```diff
@@ -99,7 +85,7 @@ private void UpdateMouseRotation(Vector3 mouseDeltaRotation)
                 timeoutTimer = 0.0f;
             }
 
-            transform.Rotate(mouseDeltaRotation, Space.World);
+            transform.Rotate(mouseDeltaRotation, Space.World); //  Self);
```

Comment:
Is this supposed to be changed instead of commented?

Reply:
good catch. thanks! it got commented while i was confirming the behavior.

Reply:
fixed